gears/src/types/base/send.rs

```diff
@@ -101,3 +101,15 @@ impl FromStr for SendCoins {
         Ok(Self::new(coins)?)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
```

Comment:
This was a bug. We should not have derived Default for SendCoins since it breaks [the condition](https://github.com/rumos-io/gears/blob/c0b6d417ddc3caf363990e49012a40e2ea1f3894/gears/src/types/base/send.rs#L28) that a valid `SendCoins` contains at least one coin.

Reply:
fixed